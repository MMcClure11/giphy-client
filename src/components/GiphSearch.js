import React from 'react'

const GiphSearch = () => {
  return (
    <div>
      This is the GiphSearch
    </div>
  )
}

export default GiphSearch
