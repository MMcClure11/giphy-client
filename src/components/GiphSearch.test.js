import React from 'react'
import Enzyme, { shallow } from 'enzyme'
import EnzymeAdapter from 'enzyme-adapter-react-16'

import GiphSearch from './GiphSearch'

Enzyme.configure({ adapter: new EnzymeAdapter() })

describe('GiphSearch', () => {
  const giphSearch = shallow(<GiphSearch />)

  it('renders properly', () => {
    expect(giphSearch).toMatchSnapshot()
  })

  // describe('when typing into the search input', () => {
  //   const query = 'puppies'

  //   beforeEach(() => {
  //     giphSearch.find('.input-query').simulate('change', { target: { value: query } })
  //   })

  //   it('updates the person in `state`', () => {
  //     expect(giphSearch.state().query).toEqual(query)
  //   })
  // })
  //Turns out you cannot call .state() on a shallow render for anything other than a class component
  //Stack Overflows say that you shouldn't really be worried about testing state
  //but focused on its render effects
})