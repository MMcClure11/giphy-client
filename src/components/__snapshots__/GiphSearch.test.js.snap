// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GiphSearch renders properly 1`] = `ShallowWrapper {}`;
